docs/deploy_local.md

```diff
@@ -51,6 +51,7 @@ export BOND_MEDIATOR_CONTRACT=${BondMediator}
 export BOND_FACTORY_CONTRACT=${BondFactory}
 export COLLATERAL_TOKENS_CONTRACT=${BitToken}
 export TREASURY_ADDRESS=${Treasury}
+export DAO_METADATA=${metadataString}
```

Comment:
issue: Name variable `DaoMetadata` as it is only for the Dao and `String` is both inconsistent and redundant.

_Description for these variables are done further up in the readme_